CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 3.14.0 (2020-03-30)
+
+- Added the `enablePointerEvents` option to `SkyOverlayConfig` which enables click events to pass through the overlay's backdrop. [#148](https://github.com/blackbaud/skyux-core/pull/148)
+- Added the `backdropClick` event to the `SkyOverlayInstance` to allow overlay instances to listen for backdrop click events. [#148](https://github.com/blackbaud/skyux-core/pull/148)
+- Added the `isTargetAboveElement()` method to the `SkyCoreAdapterService` to allow SKY UX components to detect if an event target element is contained within another element. [#148](https://github.com/blackbaud/skyux-core/pull/148)
```

Comment:
I'd delete `the` before `SkyCoreAdapterService`. (Alternately, we could add `service` as a noun after `SkyOverlayInstance`, but that seems redundant.)